DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -161,15 +161,28 @@ export class SearchSortFilterAndPaginate {
 
   static updateResultCount(count: number): void {
     const resultCount = <HTMLSpanElement>document.getElementById('results-count');
-    const oldResultCountMessage = resultCount.innerHTML;
     resultCount.hidden = false;
     resultCount.setAttribute('aria-hidden', 'false');
     let newResultCountMessage = count === 1 ? '1 matching result' : `${count.toString()} matching results`;

```

Comment:
This can go in the new function.

Reply:
Sorry, which lines specifically? Just 166: let newResultCountMessage... ?

Reply:
Think so yep. You're only calculating newResultCoundMessage to pass it into a function right? Which means you can just calculate it in the function.